### DISCUSSION TITLE
thrustmaster modena 360 steering wheel for first generation Xbox

### DISCUSSION THREAD

**Discussion by breteur** [cathery/sys-con#128] [Controller requests]
**Description of the controller**
[](https://images-na.ssl-images-amazon.com/images/I/61CBKTDVYFL._AC_SL1500_.jpg) 

thrustmaster modena 360 steering wheel for first generation Xbox.  Hello.  I found my old Xbox steering wheel.  my child would like to play Mario kart and rally game with the steering wheel.  is it possible to use this steering wheel on the Nintendo switch?  it is planned for the first generation Xbox.  Thank you

Thx

**Additional context**
Add any other context about the controller request here.

**Comment by cathery** [comment]
Try it?

**Comment by breteur** [comment]
Nothing happens.  the steering wheel is calibrated, the buttons are functional but I can't add the steering wheel to the switch menu.

**Comment by breteur** [comment]
[pictures](https://photos.app.goo.gl/FKB2bfPYtDvYqBve9)

**Comment by cathery** [comment]
As far as I know the thrustmaster wheels are usually HID devices. But maybe it isn't? Can you run this app on your switch, plug in your steering wheel, press Y and send me a screenshot? https://files.catbox.moe/re60qo.rar

**Comment by breteur** [comment]
Thx. the screenshot 
https://photos.app.goo.gl/FKB2bfPYtDvYqBve9

**Comment by cathery** [comment]
Seems like a generic HID device. sys-con doesn't support generic HID devices yet, sorry.

**Comment by breteur** [comment]
It's strange. It is the official steering wheel, however. Thank you for your answer. Continue your work it's great. Have a good day

**Comment by breteur** [comment]
I have one last question for you.  Do you have any idea what I could do with this old steering wheel?  Do you have another option to use it on the Nintendo Switch?  Thank you

**Comment by cathery** [comment]
You could look into hid-mitm, but it's unmantained and broken on latest Atmosphere.

**Comment by breteur** [comment]
Thk a lot ;)
